x/evm/handler.go

```diff
@@ -22,6 +22,9 @@ func NewHandler(server types.MsgServer) sdk.Handler {
 		case *types.MsgEthereumTx:
 			// execute state transition
 			res, err := server.EthereumTx(sdk.WrapSDKContext(ctx), msg)
+			if err == nil {
+				SetGasInContext(ctx, res.GasUsed)
```

Comment:
Up to there, we are sure that the gasMeter has the correct value.

Do you think it is enough @fedekunze ?

Reply:
I have check with simple sending transaction, it will returns a gasInfo structure with 21000 gasUsed which is the correct value.

please confirm @fedekunze